[36mATRI[0m | 03-08 10:52:09 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f5d27bec040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f5d284724f0>
         │    └ <function MultiCommand.invoke at 0x7f5d27bf00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f5d2fd5f370>
           │               │       │       └ <function Command.invoke at 0x7f5d27becaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f5d2fd5f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f5d1eb179d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f5d2fd5f370>
           │   │      │    └ <function run at 0x7f5d1ec428b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f5d27be88b0>
           └ <click.core.Context object at 0x7f5d2fd5f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f5d1ec32670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f5d2fda0940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f5d2fece430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f5d25ab0790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f5d25aaa310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f5d1e426940>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f5d25aaa1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f5d1e426940>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f5d25aaa280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f5d1e426940>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f5d2fda0940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f5d2fece430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-08 10:52:10 | ERROR plugin.manager >> Failed to import "admin_tools"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f5d27bec040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f5d284724f0>
         │    └ <function MultiCommand.invoke at 0x7f5d27bf00d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f5d2fd5f370>
           │               │       │       └ <function Command.invoke at 0x7f5d27becaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f5d2fd5f370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f5d1eb179d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f5d2fd5f370>
           │   │      │    └ <function run at 0x7f5d1ec428b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f5d27be88b0>
           └ <click.core.Context object at 0x7f5d2fd5f370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f5d1ec32670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f5d2fda0940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f5d2fece430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f5d25ab0790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f5d25aaa310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f5d1e426940>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f5d25aaa1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f5d1e426940>
                  │    │           │         └ 'admin_tools'
                  │    │           └ 'admin_tools'
                  │    └ <function PluginManager.load_plugin at 0x7f5d25aaa280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f5d1e426940>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f5d2fda0940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.admin_tools'
           │          └ <function _gcd_import at 0x7f5d2fece430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.admin_tools' from './src/plugins/admin_tools/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/admin_tools/__init__.py", line 8, in <module>
    from http_ import get_url

ModuleNotFoundError: No module named 'http_'
[36mATRI[0m | 03-08 10:52:16 | WARNING nonebot >> Duplicated prefix rule "/lottery"
[36mATRI[0m | 03-08 10:52:16 | WARNING nonebot >> Duplicated prefix rule "!lottery"
[36mATRI[0m | 03-08 10:52:16 | WARNING nonebot >> Duplicated prefix rule "！lottery"
[36mATRI[0m | 03-08 10:58:23 | WARNING nonebot >> OneBot V11 | WebSocket for Bot 507346814 closed by peer
[36mATRI[0m | 03-08 10:58:38 | WARNING nonebot >> Duplicated prefix rule "/lottery"
[36mATRI[0m | 03-08 10:58:38 | WARNING nonebot >> Duplicated prefix rule "!lottery"
[36mATRI[0m | 03-08 10:58:38 | WARNING nonebot >> Duplicated prefix rule "！lottery"
[36mATRI[0m | 03-08 10:58:48 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f5458aa6040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f545932c4f0>
         │    └ <function MultiCommand.invoke at 0x7f5458aaa0d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f5460c17370>
           │               │       │       └ <function Command.invoke at 0x7f5458aa6af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f5460c17370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f544f9d19d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f5460c17370>
           │   │      │    └ <function run at 0x7f544fafc8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f5458aa28b0>
           └ <click.core.Context object at 0x7f5460c17370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f544faeb670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f5460c58940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f5460d86430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f545696a790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f5456964310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f544f2e1820>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f54569641f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f544f2e1820>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f5456964280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f544f2e1820>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f5460c58940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f5460d86430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-08 11:01:17 | WARNING nonebot >> OneBot V11 | WebSocket for Bot 507346814 closed by peer
